src/main/webapp/electionlist.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>gVote</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
+
+    <!--Bootstrap Libraries-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <a href="index.html">
+      <div id="header-bar">
+        <p id="header-text">gVote</p>
+      </div>
+    </a>
+
+    <div class="container" id="content">
+      <div class="row page-title-text" id="dropdown-title">
+        <p class="col-lg-8 col-lg-offset-2 col-md-8 col-md-offset-2 col-sm-8 col-sm-offset-2">
+          <b>To explore the upcoming elections you can vote in, select the state you live in:</b>
+        </p> 
+      </div>
+      <div class="row" id="state-dropdown">
+        <select id="select-state" name="states" onchange="listElections()">
```

Comment:
Do we have this method already? If not maybe omit the onchange until we add the corresponding js function.